data/World/Overworld.json

```diff
@@ -736,7 +745,9 @@
             "ToT Light Arrows Cutscene": "is_adult and can_trigger_lacs"
         },
         "exits": {
-            "ToT Entrance": "True",
+            "ToT Entrance": "
+                (open_forest != 'closed' or starting_age == 'child' or here(is_child) or
```

Comment:
I think there should be a comment here explaining why these logical requirements exist.